### PR TITLE
BBIP 1: Using conda environment file

### PR DESCRIPTION
**Pull request by sbenthall**
See issue #248

### CODE REVIEW

**Comment by sbenthall** [comment]
Most of these commits (everything except for 76ecb8b) are actually having to do with the website.

This is a little confusing to me, since the website files are loaded from the gh-pages branch.

But it looks like the files from the gh-pages branch are also before stored in master. Commits to one branch are getting merged to the other.

Some research on the Internet suggests that this is desirable to some people. I'm confused by it myself and am not sure why this is happening. Though it's possible I set it up this way accidentally.

File: README.md [hunk 1]
```diff
@@ -22,20 +22,9 @@ Run the following commands:
 
 ```bash
 git clone https://github.com/nllz/bigbang.git
```
Comment (npdoty) [inline README.md]: Users should be cloning from the `datactive` repository, not `nllz/`, right?

File: README.md [hunk 2]
```diff
@@ -22,20 +22,9 @@ Run the following commands:
...
-### pip installation
-
-Run the following commands:
-
-```bash
-git clone https://github.com/sbenthall/bigbang.git
-# optionally create a new virtualenv here
-pip install -r requirements.txt
-python setup.py develop
+conda create -f bigbang environment.yaml
```
Comment (npdoty) [inline README.md]: From documentation, it seems like `conda env create -f environment.yaml` also works. Is there any reason we need to specify the env name in this command?

Comment (sbenthall) [inline README.md]: No, this is an error in the PR. Thanks for catching it! I'll fix it.

**Comment by sbenthall** [comment]
I've addressed your comments on the README, @npdoty 

I've also merged in other commits from master, making this pull request up to date

**Comment by npdoty** [comment]
I'm confused why the commits/files-changed views show huge numbers of changes if those changes are already present in master. The changes to the README seem fine, but it's unsettling to see a thousand lines and 18 other files as included.

**Pull request closed without merging**